sci-biology/fasta/files/fasta-36.3.8i-musl-build-fix.patch

```diff
@@ -0,0 +1,58 @@
+https://github.com/wrpearson/fasta36/pull/64
+From: Brahmajit Das <brahmajit.xyz@gmail.com>
+Date: Fri, 30 Aug 2024 11:01:16 +0000
+Subject: [PATCH 1/1] Fix building on musl libc
+
```

Comment:
This is a bot long, please drop this header and add an url to the upstream commit.